src/domain/audio/AudioInput.ts

```diff
@@ -46,7 +46,6 @@ class AudioInput {
 
     #_readyRead = new Signal();
 
-
```

Comment:
It would be good to reinstate these removed blank lines because they help delineate between the different types of methods.